src/components/footer/_macro-options.md

```diff
@@ -50,3 +50,19 @@
 | Name      | Type                                                        | Required | Description                 |
 | --------- | ----------------------------------------------------------- | -------- | --------------------------- |
 | itemsList | array`<ListItem>` [_(ref)_](/foundations/typography/#lists) | true     | A list of links for the row |
+
+## PoweredBy
+
+| Name               | Type                 | Required | Description                                                                             |
+| ------------------ | -------------------- | -------- | --------------------------------------------------------------------------------------- |
+| classes            | string               | false    | Classes to be added. Helpful to add a margin utility class to control spacing.          |
```

Comment:
Dont think we need to add "Helpful to add a margin utility class to control spacing." Its not the sort of thing we have put in the description of other params